compiler/rustc_builtin_macros/src/test.rs

```diff
@@ -252,6 +252,9 @@ pub fn expand_test_or_bench(
                                         "ignore",
                                         cx.expr_bool(sp, should_ignore(&cx.sess, &item)),
                                     ),
+                                    // ignore_message: None
+                                    #[cfg(not(bootstrap))]
```

Comment:
The compiler likely *should* always produce/permit the field, I don't think you need to gate it on non-bootstrap here.